calcite/src/main/java/husky/sql/calcite/table/SimplePartsTable.java

```diff
@@ -0,0 +1,145 @@
+package husky.sql.calcite.table;
+
+
+import org.apache.calcite.DataContext;
+import org.apache.calcite.linq4j.Enumerable;
+import org.apache.calcite.linq4j.Linq4j;
+import org.apache.calcite.rel.type.RelDataType;
+import org.apache.calcite.rel.type.RelDataTypeFactory;
+import org.apache.calcite.rel.type.RelDataTypeFactory.Builder;
+import org.apache.calcite.rel.type.RelProtoDataType;
+import org.apache.calcite.schema.ScannableTable;
+import org.apache.calcite.schema.impl.AbstractTable;
+import org.apache.calcite.sql.type.SqlTypeName;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.ArrayList;
+
+//import jdk.nashorn.internal.parser.JSONParser;
+
+
+public class SimplePartsTable extends AbstractTable implements ScannableTable{
```

Comment:
No need to implement the `ScannableTable` interface. This is supposed to be used by Husky right?